core/java/android/provider/Settings.java

```diff
@@ -3160,6 +3160,12 @@ public boolean validate(String value) {
          */
         public static final int END_BUTTON_BEHAVIOR_DEFAULT = END_BUTTON_BEHAVIOR_SLEEP;
 
+        /**
+        * List of Apps hidden from recents
+        * @hide
+        */
+        public static final String HIDE_FROM_RECENTS_LIST = "hide_from_recents_list";
```

Comment:
Why is this placed here? Should be on end where last setting was added by us.

Reply:
can add at end no prob